lib/src/emoji_cell.dart

```diff
@@ -99,16 +101,39 @@ class EmojiCell extends StatelessWidget {
     );
   }
 
-  /// Build and display Emoji centered of its parent
-  Widget _buildEmoji() {
-    final style = TextStyle(
+  static String? _getPlatformDefaultEmojiFont() {
+    if (Platform.isIOS || Platform.isMacOS) {
```

Comment:
This won't work for web. In order to support web we always need to test for web first.
```suggestion
    if (kIsWeb) return null;
    else if (Platform.isIOS || Platform.isMacOS) {
```

Reply:
I'm not very familiar with Flutter on the web. It makes sense that the user could be using a browser on iOS or MacOS and that `Platform.isIOS` would return true in such a case. However, I think that should be fine since Apple's emoji font would be present.

Can you help me understand what the issue is?

Reply:
`Platform.isIOS` or `Platform.isMacOS` does crash if someone compiles it for web. Platform is not resolvable for web, because of that the `kIsWeb` is not from Platform but a constant, which we need to check first before we can check for any desktop or mobile platform.